### PR TITLE
Sarah content edits

### PR DESCRIPTION
**Pull request by sarahsimpson09**
These are the rest of the pull requests I submitted yesterday, all combined into one.

### CODE REVIEW

File: budget/funding-request/index.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
 
     <div class="block block__flush-top block__sub">
         <p class="h3">
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit.
+            As required by law, each fiscal quarter the Federal Reserve transfers to the CFPB the amount of funding formally requested by the Bureau director in order to carry out Bureau activities as outlined in our budget. Once the funds are transferred, the Bureau receives an acknowledgement letter from the Federal Reserve Board.
```
Comment (jimmynotjim) [inline budget/funding-request/index.html]: We need to add some semantic line breaks to reduce hoz scrolling here.

File: the-bureau/about-rich-cordray/index.html [hunk 2]
```diff
@@ -76,7 +76,7 @@ <h1 class="h4 u-mb5">Downloads</h1>
             <div class="content-l_col
                         content-l_col-1-2">
                 <h1 class="h4 u-mb5">
-                    Request speaking info
+                    Request the director's speaking info
```
Comment (jimmynotjim) [inline the-bureau/about-rich-cordray/index.html]: `'` should be a `&rsquo;` in `director's`

File: the-bureau/about-rich-cordray/index.html [hunk 3]
```diff
@@ -146,12 +146,13 @@ <h1 class="h2">
...
                     <figure class="figure__bordered">
-                        <a class="video-poster" href="#">
+                        <a class="jump-link jump-link__external-link"
+                       href="https://www.youtube.com/user/cfpbvideo/search?query=Cordray%27s+Corner">
                             <img src="cordrays_corner.jpg" alt="">
                         </a>
                     </figure>
                     <p class="short-desc short-desc__large">
-                        View videos on Director Richard Cordray discussing various relevant fiancial topics and how the bureau is involved.
+                        View videos of Director Richard Cordray discussing various relevant fiancial topics and how the bureau is involved.
```
Comment (jimmynotjim) [inline the-bureau/about-rich-cordray/index.html]: Add a semantic break after "topics"

File: the-bureau/about-rich-cordray/index.html [hunk 4]
```diff
@@ -168,7 +169,7 @@ <h1 class="h2">
                         <img src="../obama_cordray.jpg" alt="">
                     </figure>
                     <p class="short-desc short-desc__large">
-                        In January of 2012, President Barack Obama appointed Rich Cordray to be the first Director of the CFPB.
+                        In January of 2012, President Barack Obama appointed Richard Cordray to be the first Director of the CFPB.
```
Comment (jimmynotjim) [inline the-bureau/about-rich-cordray/index.html]: Add a semantic break after "first"

File: the-bureau/about-steve-antonakes/index.html [hunk 5]
```diff
@@ -72,7 +72,7 @@ <h1 class="h4 u-mb5">
             </div>
             <div class="content-l_col content-l_col-1-2">
                 <h1 class="h4 u-mb5">
-                    Request speaking info
+                    Request the deputy director's speaking info
```
Comment (jimmynotjim) [inline the-bureau/about-steve-antonakes/index.html]: This `'` should also be `&rsquo;` or `’`

File: the-bureau/about-rich-cordray/index.html [hunk 6]
```diff
@@ -127,11 +127,11 @@ <h2 class="h5">
                         Calendar
                     </h2>
                     <p class="short-desc">
-                        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Expedita id minima ut nulla.
+                        See how the director spends his time working for consumers.
                     </p>
                     <a class="jump-link jump-link__right"
                        href="{{ vars.path + 'leadership-calendar/' }}">
-                        View Director Cordray's calendar
+                        View the director's public calendar
```
Comment (jimmynotjim) [inline the-bureau/about-rich-cordray/index.html]: This ' should also be &rsquo; or ’

**Comment by sarahsimpson09** [comment]
Ok, I've made these updates.

**Comment by jimmynotjim** [comment]
:+1: Thanks.

**Pull request merged**